includes/class-main.php

```
@@ -18,988 +18,993 @@
 
-	/**
-	 * Enqueues scripts for performance optimization.
-	 *
-	 * @since 1.0.0
-	 */
-	public function enqueue_scripts() {
-		if ( is_admin_bar_showing() ) {
-			$asset_file = WPPO_PLUGIN_PATH . 'build/main.asset.php';
-			$resolved   = wp_normalize_path( realpath( $asset_file ) );
+			wp_set_script_translations( 'performance-optimisation-script', 'performance-optimisation' );
+		}
```

PHPMD (2.15.0)

[warning] The method admin_enqueue_scripts() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)